src/test/java/com/google/api/codegen/testdata/discogapic/java/java_simplecompute.v1.json.baseline

```diff
@@ -5572,34 +5661,75 @@ import javax.annotation.Generated;
-          io.grpc.MethodDescriptor.create(
-              io.grpc.MethodDescriptor.MethodType.UNARY,
-              "google.simplecompute.v1.Addresses/listAddresses",
-              io.grpc.protobuf.ProtoUtils.marshaller(ListAddressesHttpRequest.getDefaultInstance()),
-              io.grpc.protobuf.ProtoUtils.marshaller(AddressList.getDefaultInstance())));
+          ApiMethodDescriptor.create(
+              "compute.addresses.list",
+              ListAddressesHttpRequest.getDefaultInstance(),
+              AddressList.getDefaultInstance(),
+              "{project}/regions/{region}/addresses",
+              new HashSet<String>(
+                  Arrays.asList(
+                      "project",         "region"
+                      )
+                  ),
+              new HashSet<String>(
+                  Arrays.asList(
+                      "filter",         "maxResults",         "orderBy",         "pageToken"
+                      )
+                  ),
+              new ApiMessageHttpRequestBuilder(),
+              HttpMethod.GET));
   private static final UnaryCallable<GetAddressHttpRequest, Address> directGetAddressCallable =
       HttpJsonCallableFactory.createDirectCallable(
-          io.grpc.MethodDescriptor.create(
-              io.grpc.MethodDescriptor.MethodType.UNARY,
-              "google.simplecompute.v1.Addresses/getAddress",
-              io.grpc.protobuf.ProtoUtils.marshaller(GetAddressHttpRequest.getDefaultInstance()),
-              io.grpc.protobuf.ProtoUtils.marshaller(Address.getDefaultInstance())));
+          ApiMethodDescriptor.create(
+              "compute.addresses.get",
+              GetAddressHttpRequest.getDefaultInstance(),
+              Address.getDefaultInstance(),
+              "{project}/regions/{region}/addresses/{address}",
+              new HashSet<String>(
+                  Arrays.asList(
+                      "address",         "project",         "region"
+                      )
+                  ),
+              new HashSet<String>(
+                  ),
+              new ApiMessageHttpRequestBuilder(),
+              HttpMethod.GET));
   private static final UnaryCallable<DeleteAddressHttpRequest, Operation> directDeleteAddressCallable =
       HttpJsonCallableFactory.createDirectCallable(
-          io.grpc.MethodDescriptor.create(
-              io.grpc.MethodDescriptor.MethodType.UNARY,
-              "google.simplecompute.v1.Addresses/deleteAddress",
-              io.grpc.protobuf.ProtoUtils.marshaller(DeleteAddressHttpRequest.getDefaultInstance()),
-              io.grpc.protobuf.ProtoUtils.marshaller(Operation.getDefaultInstance())));
+          ApiMethodDescriptor.create(
+              "compute.addresses.delete",
+              DeleteAddressHttpRequest.getDefaultInstance(),
+              Operation.getDefaultInstance(),
+              "{project}/regions/{region}/addresses/{address}",
+              new HashSet<String>(
+                  Arrays.asList(
```

Comment:
I think `Sets.newHashSet` would be nicer

Reply:
Done.